lightning_hpo/controllers/sweep.py

```diff
@@ -119,23 +119,32 @@ def delete_sweep(self, config: DeleteSweepConfig):
                     sweep = sweep.collect_model()
                     del self.r[config.name]
                 self.db.delete(sweep)
+                for tensorboard in self.db.select_all(TensorboardConfig):
```

Comment:
I believe this should delete tensorboard instead and every resources associated to the Sweep.